src/lesson/lesson03.js

```diff
@@ -1418,11 +1499,60 @@ function Lesson03() {
       ),
       "",  // initialCode
       Constants.Lesson03.SUCCESS_MESSAGE,
-      null
+      40
```

Comment:
This limit is not enough to force the use of conditionals. A possible solution is `FLGRFRPLFRGLFLPRFLGRFRPLFRGLFLP`. I think we'll need "require_conditionals" here too.